### PR TITLE
add high level development instructions

### PR DESCRIPTION
**Pull request by jdkandersson**

### CODE REVIEW

**Comment by jdkandersson** [comment]
Just found some more documentation I want to add, please hold review.

**Comment by jdkandersson** [comment]
Alright, good for review now

**Comment by mthaddon** [comment]
I think this is already covered by https://charmhub.io/indico/docs/contributing - can you take a look and confirm? The nature of documentation like this is still something we're in discussions with the documentation team about, but our current understanding is that this kind of content should be covered in the discourse-based detailed documentation that's rendered on charmhub.

**Comment by jdkandersson** [comment]
> I think this is already covered by https://charmhub.io/indico/docs/contributing - can you take a look and confirm? The nature of documentation like this is still something we're in discussions with the documentation team about, but our current understanding is that this kind of content should be covered in the discourse-based detailed documentation that's rendered on charmhub.

Ah I see, do you think it is worth linking to the charmhub developer docs from the README file? The `juju deploy ./indico-operator_ubuntu-20.04-amd64.charm` command on charmhub is missing the resources (which it says in the comment above it as well), how do updates to that documentation work? I'm happy to add that part. The charmhub docs also seem to be missing the instructions for building the Docker images and how to get them into microk8s from Docker

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Ignore everything
```
Comment (arturo-seijas) [inline .dockerignore:1]: I think It'd be a good idea to split this dockerignore changes to another PR. I don't think they match the PR scope and they affect the build process.

**Review by arturo-seijas — CHANGES_REQUESTED** [review]

**Comment by mthaddon** [comment]
> Ah I see, do you think it is worth linking to the charmhub developer docs from the README file? The `juju deploy ./indico-operator_ubuntu-20.04-amd64.charm` command on charmhub is missing the resources (which it says in the comment above it as well), how do updates to that documentation work? I'm happy to add that part. The charmhub docs also seem to be missing the instructions for building the Docker images and how to get them into microk8s from Docker

The readme does already link to that - `For further details, [see here](https://charmhub.io/indico/docs)`. I can explain how to edit the discourse docs when we catch up tomorrow.

File: .dockerignore (hunk 1 above)
Comment (mthaddon) [reply .dockerignore:1]: I agree on this.

**Comment by mthaddon** [comment]
Closing out this pull request as discussed with David - changes will be integrated into the discourse-based documentation.

**Pull request closed without merging**